src/symbol/grounding_set.cc

```diff
@@ -308,6 +310,8 @@ from_xml( xmlNodePtr root ){
           _groundings.push_back( new Constraint( l1 ) );
         } else if ( xmlStrcmp( l1->name, ( const xmlChar* )( "container" ) ) == 0 ){
           _groundings.push_back( new Container( l1 ) );
+        } else if ( xmlStrcmp( l1->name, ( const xmlChar* )( "container_type" ) ) == 0 ){
```

Comment:
Hmmm, is this being included twice? Looks like lines 287-288 include the "rule_container_type" already. Lines 325-326 include "container_type".